modules/402-ingress-nginx/openapi/config-values.yaml

```
@@ -4,7 +4,7 @@ properties:
     default: "1.9"
```

let's change the default to 1.10. Why not?
```suggestion
    default: "1.10"
```